.github/workflows/mirror-calico.yml

```diff
@@ -0,0 +1,27 @@
+name: Sync GHCR Calico images with Docker Hub
+on:
+  schedule:
+    # run every 12h
+    - cron:  '0 */12 * * *'
+  workflow_dispatch:
+
+jobs:
+  get-docker-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Login to GitHub Container Registry (ghcr)
+        run: echo ${{ secrets.GHCR_PASSWORD }} | docker login ghcr.io -u ${{ secrets.GHCR_USERNAME }} --password-stdin
+      - name: Fetch latest Calico release
+        id: fetch-latest-release
+        run: |
+          set -ex
+
+          git clone https://github.com/projectcalico/calico calico
+          # get latest version
+          version=$(git -C "$_" tag | sort -V | tail -1)
+
+          pushd .github/workflows/
+          chmod u+x ./mirror-to-ghcr.sh ./mirror-calico.sh
```

Comment:
This should not be necessary, right?

Reply:
That's right, scripts already hold `x` flag: 
```
$ ls -ali .github/workflows/mirror-*.sh
73843545 -rwxr-xr-x 1 mathieu mathieu  439 Dec  7 14:22 .github/workflows/mirror-calico.sh
73842802 -rwxr-xr-x 1 mathieu mathieu 1021 Dec  7 14:18 .github/workflows/mirror-to-ghcr.sh
```